packages/snip20/Readme.md

```diff
@@ -12,9 +12,14 @@ Or you can call the individual function for each Handle message to generate the
 
 Example:
 
-```ignore
+```rust
+# use cosmwasm_std::{Uint128, StdError, StdResult, CosmosMsg, Response};
+# use secret_toolkit_snip20::{transfer_msg};
+#
+# fn main() -> StdResult<()> {
```

Comment:
Is `main()` necessary?
If so, can you un-indent the rest of the code to match the rest of the docstrings?